src/objects/api/v2/filters.py

```diff
@@ -124,7 +124,9 @@ def filter_data_icontains(self, queryset, name, value: str):
         return queryset.extra(where=[where_str], params=[value])
 
     def filter_date(self, queryset, name, value: date_):
-        return queryset.filter_for_date(value)
+        """actual filtering is performed in the viewset"""
```

Comment:
What does this do? From what I can see all this does is return the queryset. Can't we just remove this?

Reply:
It's added to the filterset to generate the OAS with correct query params
But the actual filtering is moved to the viewset now, because it should be done before filtering on other query params